src/js/data_storage.js

```diff
@@ -15,6 +15,7 @@ const API_VERSION_1_42 = '1.42.0';
 const API_VERSION_1_43 = '1.43.0';
 const API_VERSION_1_44 = '1.44.0';
 const API_VERSION_1_45 = '1.45.0';
+const API_VERSION_1_46 = '1.46.0';
```

Comment:
```suggestion
```

Reply:
Gotcha. Now that the FW API version has reverted to 1.45 this is the thing to do.
Fixed